gui/wxpython/gui_core/gselect.py

```diff
@@ -3086,7 +3073,7 @@ def _onMapClickHandler(self, event):
                 self.OnClose()
             else:
                 self.catsField.SetValue(self._vectorSelect.GetLineStringSelectedCats())
-        else:
+        else:  # noqa: PLR5501
```

Comment:
Also this could be solved with two _elif_ s, no?

Reply:
Reformulated the complete conditions